packages/flutter/lib/src/material/material.dart

```diff
@@ -91,10 +91,11 @@ abstract class MaterialInkController {
 /// of material cannot be printed beyond the bounds of the material.
 ///
 /// If the layout changes (e.g. because there's a list on the paper, and it's
-/// been scrolled), a LayoutChangedNotification must be dispatched at the
-/// relevant subtree. (This in particular means that Transitions should not be
-/// placed inside Material.) Otherwise, in-progress ink features (e.g., ink
-/// splashes and ink highlights) won't move to account for the new layout.
+/// been scrolled), a [LayoutChangedNotification] must be dispatched at the
+/// relevant subtree. This in particular means that transitions (e..g.
+/// [SlideTransition]) should not be placed inside Material. Otherwise,
```

Comment:
Scaffold creates a Material widget, so don't most transitions occur "inside Material"?

Reply:
Yup. I clarified this a bit to say that it's specifically a problem if you move ink features around.